test/should-compile/ap_ints.sea

```diff
@@ -0,0 +1,12 @@
+/////////////////////////////////////////////
+// ap_ints.sea
```

Comment:
You need to add a test to the test_programs.ml file in order to get this test to run.